Makefile

```diff
@@ -260,8 +260,11 @@ versioncontrol_%:
 .PHONY: linksharing_%
 linksharing_%:
 	GOOS=$(word 2, $(subst _, ,$@)) GOARCH=$(word 3, $(subst _, ,$@)) COMPONENT=linksharing $(MAKE) binary
+.PHONY: auto-updater_%
+auto-updater_%:
+	GOOS=$(word 2, $(subst _, ,$@)) GOARCH=$(word 3, $(subst _, ,$@)) COMPONENT=auto-updater $(MAKE) binary	
 
-COMPONENTLIST := bootstrap certificates gateway identity inspector linksharing satellite storagenode uplink versioncontrol
+COMPONENTLIST := bootstrap certificates gateway identity inspector linksharing satellite storagenode uplink versioncontrol auto-updater
```

Comment:
Put this in alphabetical order before `bootstrap` please.

Reply:
done